tpcx_bb/queries/q02/tpcx_bb_query_02_sql.py

```diff
@@ -81,6 +83,9 @@ def main(data_dir, client, bc, config):
         LIMIT {q02_limit}
     """
     result = bc.sql(last_query)
+    result["item_sk_2"] = q02_item_sk
```

Comment:
We'd probably eventually want to push this into the SQL statement again like in the original Spark SQL/HQL queries.

Reply:
Let me explore using this version.